File: C++/Linked-List/Display.cpp
#include <iostream>
using namespace std;

class Node
{
public:
    int data;
    Node *next;
};

Node *create(int A[], int n)
{
    Node *t;
    Node *last;

    Node *head = new Node;
    head->data = A[0];
    head->next = nullptr;
    last = head;

    for (int i = 1; i < n; i++)
    {
        t = new Node;
        t->data = A[i];
        t->next = nullptr;
        last->next = t;
        last = t;
    }

    return head;
}

void display(Node* p)
{
    while (p != NULL)
    {
        cout << " " << p->data;
        p = p->next;
    }
}

int main()
{
    cout << "Code Execution started...!\n";

    int A[] = {10, 70, 50, 39, 500, 1729, 2000};
    int n = sizeof(A) / sizeof(A[0]);

    Node *ptrHead = create(A, n);

    display(ptrHead);

    cout << endl;

    return 0;
}

File: Java/Linked-List/Display.java

class LinkedList {

    Node head;

    static class Node {
        int data;
        Node next;

        Node(int d) {
            data = d;
            next = null;
        }
    }

    public static void insert(LinkedList list, int data) {

        Node new_node = new Node(data);
        new_node.next = null;

        if (list.head == null) {
            list.head = new_node;
        } else {

            Node last = list.head;
            while (last.next != null) {
                last = last.next;
            }

            last.next = new_node;
        }
    }

    public static void printList(LinkedList list) {

        Node currNode = list.head;

        System.out.print("LinkedList: ");

        while (currNode != null) {

            System.out.print(currNode.data + " ");
            currNode = currNode.next;
        }
    }

    public static void main(String[] args) {
        System.out.println("Hello World \n");

        LinkedList list = new LinkedList();

        insert(list, 90);

        printList(list);
    }
}